tardis/montecarlo/base.py

```diff
@@ -214,24 +199,18 @@ def _initialize_geometry_arrays(self, model):
     def _initialize_packets(
         self, temperature, no_of_packets, iteration, radius, time_explosion
     ):
-        # the iteration is added each time to preserve randomness
-        # across different simulations with the same temperature,
-        # for example. We seed the random module instead of the numpy module
-        # because we call random.sample, which references a different internal
-        # state than in the numpy.random module.
-        seed = self.seed + iteration
-        rng = np.random.default_rng(seed=seed)
-        seeds = rng.choice(MAX_SEED_VAL, no_of_packets, replace=True)
```

Comment:
This is possibly a different order to what is in the reseed method

Reply:
Yeah though I had been careful but seems like I missed something here. Anyway I have restructured things now, and the order is exactly the same now. (Now my local SDEC plot and the SDEC plot in the documentation page are matching exactly)